modules/custom/openy_programs_search/config/install/openy_programs_search.settings.yml

```diff
@@ -0,0 +1,8 @@
+default_locations: []
+client_id: ''
+domain: ''
+base_url: ''
+registration_path: ''
+get_schools_by_program_path: ''
+get_categories_path: ''
+get_map_categories_by_branch_path: ''
```

Comment:
@Sanchiz Should I go ahead and set these to the assumed defaults given in the form descriptions?